File: Day-1/Q4.c
#include <stdio.h>
#include <stdlib.h>

int count(long long n){
    int c = 0;
    if(n == 0)
        return 1;
    n = llabs(n);
    while(n > 0){
        c++;
        n /= 10;
    }
    return c;
}

int main(){
    long long n;
    printf("Enter a Number : ");
    scanf("%lld", &n);
    printf("No. of Digits = %d", count(n));
    return 0;
}

File: Day-1/Q4.java
import java.util.Scanner;

public class Q4{
    public static int count(long n){
        int c = 0;
        if(n == 0)
            return 1;
        n = Math.abs(n);
        while(n > 0){
            n /= 10;
            c++;
        }
        return c;
    }

    public static void main(String[] args){
        try(Scanner s = new Scanner(System.in)){
            System.out.print("Enter a Number : ");
            long n = s.nextLong();
            System.out.println("No. of Digits = " + count(n));
        }
    }
}